### PR TITLE
Created connection to db and base tables

### PR DESCRIPTION
**Pull request by AndreyShelest**

### CODE REVIEW

File: sources/db/__init__.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+class NativeCoroutinesRequestHandler(SessionMixin, RequestHandler):
+    async def get(self):
+        with self.make_session() as session:
+            count = await as_future(session.query(User).count)
+
+        self.write('{} users so far!'.format(count))
+
+
+class GenCoroutinesRequestHandler(SessionMixin, RequestHandler):
+    @coroutine
```
Comment (approximator) [inline sources/db/__init__.py]: Maybe it's better to use async everywhere and not mix it with `@coroutine`?

File: sources/db/__init__.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+"""
+
+from tornado.gen import coroutine
+from tornado.web import RequestHandler
+from tornado_sqlalchemy import as_future, SessionMixin
+
+from db.models import User
+
+
+class NativeCoroutinesRequestHandler(SessionMixin, RequestHandler):
```
Comment (approximator) [inline sources/db/__init__.py]: These handlers are part of the server.
Why did you put them here into `__init__`?

File: sources/db/models/user.py [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+
+class User(BASE_MODEL):
+    __tablename__ = 'users'
+    id = Column(Integer, nullable=False, primary_key=True, autoincrement=True)
+    username = Column(UnicodeText(40), unique=True)
+    pwd_hash = Column(UnicodeText(80), unique=False)
+
+    # currency code according to ISO 4217
+    currency = Column(UnicodeText(3), nullable=False, default="USD", unique=False)
```
Comment (approximator) [inline sources/db/models/user.py]: I think `currency` should be as a property of account or `wallet`, but not user :)

**Review by approximator — COMMENTED** [review]
Quite a mess with imports.
How do you even start this?

File: sources/db/__init__.py (hunk 1 above)
Comment (AndreyShelest) [reply sources/db/__init__.py]: I have got errors when used async handlers which SQlite database

**Comment by AndreyShelest** [comment]
to resolve issue with imports just add 'sources' folder to PYTHONPATH. or mark it as sources in PyCharm project configuration

File: sources/db/__init__.py (hunk 2 above)
Comment (AndreyShelest) [reply sources/db/__init__.py]: outdated

**Comment by approximator** [comment]
>  just add 'sources' folder to PYTHONPATH. or mark it as sources in PyCharm project configuration

This is a bad workaround, I think.
Moreover, there is no any README or commit message saying about this )

**Comment by approximator** [comment]
```
# curl http://127.0.0.1:8788/budget
<html><title>500: Internal Server Error</title><body>500: Internal Server Error</body></html>
```

```
sqlalchemy.exc.OperationalError: (sqlite3.OperationalError) no such table: users [SQL: 'SELECT users.id AS users_id, users.login AS users_login, users.pwd_hash AS users_pwd_hash, users.username AS users_username, users.user_pic AS users_user_pic \nFROM users\n LIMIT ? OFFSET ?'] [parameters: (1, 0)] (Background on this error at: http://sqlalche.me/e/e3q8)
09-10-2018:22:31:13.266 ERROR    [web.py:2162] 500 GET /budget (127.0.0.1) 18.49ms
```

**Review by Radeon4650 — APPROVED** [review]
Good work!

**Pull request merged**